### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by lllama** [Textualize/textual#382] [General]
Not wanting to hold @willmcgugan to anything but is there any roadmap/planned feature list somewhere? And, if not, is there any merit in having one? I'm asking purely out of selfish reasons, as the vast majority of my use cases would involve some sort of form fields, so I'm curious as to where they live on the priority list.

**Comment by ThatXliner** [comment]
https://github.com/Textualize/textual/projects/1

**Reply by lllama** [reply]
Shiny! I should learn to GitHub better.

**Reply by lllama** [reply]
Shiny new roadmap: https://textual.textualize.io/roadmap/

**Reply by WMRamadan** [reply]
@lllama Would you be able to share what kind of project you will be using the form fields for?

**Comment by midnqp** [comment]
@davep @willmcgugan A podcast from [The Changelog](https://changelog.com/) is how I heard about [Textualize](https://textualize.io/). I checked the roadmap. I just wanted to ask this candidly: if someone wanted to contribute something big to this project, what would that be? In other words, how can I help? This project seems too ✨ perfect/well-managed! Any issues or PRs that we can begin with?

**Reply by davep** [reply]
Hey @MidnQP -- thanks for your kind words. Right at the moment, ideally, the most helpful things people can do for us include:

- Test/use the docs and provide feedback
- Build applications with Textual and provide us with feedback
- Build your own widget libraries and make them available on your GitHub and/or on PyPi
- Help get the word out about Textual

That's not to say we can't/won't accept PRs to Textual itself -- that's happening plenty, but right now we're not looking for folk to take on specific tasks within the codebase.

**Reply by midnqp** [reply]
Sounds like a plan 🎉!

I'll be sure play around with this TUI framework. I'm excited. Obviously our end-goal would be to build like a full-fledged Textual App Store in the future,  as Will mentioned in the podcast. In the meantime however, I have the opportunity to master the craft, so that I can contribute something great to the  🚀ecosystem!

**Reply by WMRamadan** [reply]
@MidnQP Can you link the podcast episode where @willmcgugan mentioned a Textual App Store?
